gerbera-web/test/client/gerbera.app.spec.js

```diff
@@ -27,7 +27,7 @@ describe('Gerbera UI App', () => {
     });
 
     afterEach(() => {
-      ajaxSpy.calls.reset();
+      ajaxSpy.and.callThrough();
```

Comment:
The testing of AJAX requires 2 capabilities.

1. Mocking the `$.ajax` to avoid calling the server during unit tests
2. Permit `$.ajax` to allow jasmine-jquery fixtures to be loaded into the DOM

> You will see repeated `.and.callThrough()` which allows any subsequent ajax calls to be made.  In majority of cases this is a fixture being downloaded for the next test.